projects/addon-doc/src/components/demo/demo.template.html

```diff
@@ -154,6 +157,7 @@
                                 tuiButton
                                 type="submit"
                                 size="s"
+                                automation-id="tui-demo-button__submit-state"
```

Comment:
<img width="1179" alt="image" src="https://user-images.githubusercontent.com/12021443/211670425-51e2d978-83c4-4510-8e84-e1195db647e7.png">

need for master branch
https://github.com/Tinkoff/taiga-ui/pull/3400